indico/modules/rb/client/js/modules/roomList/RoomList.jsx

```diff
@@ -176,6 +206,12 @@ class RoomList extends React.Component {
                     <div styleName="actions">
                       {selectionMode ? (
                         <>
+                          <ResponsivePopup
+                            trigger={
+                              <Button icon={selectAllIcon} onClick={this.selectAll} circular />
+                            }
+                            content={selectAllText}
+                          />{' '}
```

Comment:
Is the space needed?
```suggestion
                          />
```

Reply:
Well, @GovernmentPlates requested some spacing and I thought that was the easiest way^^ I'd also be happy to add a CSS margin, but I really don't know where to put it. As element style attribute, add an element ID and respective style, or add a class for one element and one line..?

Reply:
I was thinking of using the in-line style for this (i.e. `style={{marginRight: '1.5em'}}`) - this would be the easiest option for spacing it out.